Fix:
error: implicit declaration of function 'usleep' [-Werror,-Wimplicit-function-declaration]
https://github.com/rwos/gti/pull/59
--- gti.c.orig	2020-02-09 07:59:44.000000000 -0600
+++ gti.c	2022-03-30 21:03:57.000000000 -0500
@@ -16,7 +16,7 @@
 #    define WIN32
 #else
      /* fileno() */
-#    define _POSIX_C_SOURCE 1
+#    define _POSIX_C_SOURCE 199506L
      /* usleep() */
 #    define _DEFAULT_SOURCE
 #endif
